src/Analyzer.Types/TemplateContext.cs

```diff
@@ -20,6 +20,16 @@ public class TemplateContext
         /// The template with resolved parameter/variable references and expressions
         /// </summary>
         public JToken ExpandedTemplate { get; set; }
+        
+        /// <summary>
+        /// Whether this template was originally a Bicep file
+        /// </summary>
+        public bool IsBicep { get; set; }
```

Comment:
Should there be a default value/is there a default value?

Reply:
I think the default value would be false for a bool. There's only one location in Analyzer.Core where TemplateContext is created that will always directly set this value as well.